### PR TITLE
add follow and user model

### PR DESCRIPTION
**Pull request by eliasreis54**
The `flutter format .` changed the style of some files. 

Does we need other models? :thinking:

### CODE REVIEW

File: lib/models/follow.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import './user.dart';
+
+class FollowModel {
```
Comment (thomashorta) [inline lib/models/follow.dart]: Not sure if we need this model.

**Review by thomashorta — COMMENTED** [review]
Are those models the server-side model (Firestore) or the app-side model? We might need to have models (or entities) that model the exact Firestore Documents/Collections and models that we actually use in the app, and some kind of Mapper/Converter classes/methods to convert between them. 

I think the models proposed are more suited to be the app-side models. For the Firestore side I think we could try something like this stackoverflow answer proposes: https://stackoverflow.com/a/51591848/2237911

**Comment by eliasreis54** [comment]
I changed the models to put ids as string, like the [reference](https://googleapis.dev/nodejs/firestore/latest/DocumentReference.html)

File: lib/models/follow.dart (hunk 1 above)
Comment (eliasreis54) [reply lib/models/follow.dart]: Okay... i removed that.

**Review by erickzanardo — APPROVED** [review]
LGTM

**Pull request merged**